src/article-editor.jsx

```diff
@@ -7,11 +7,12 @@ var Editor = require("./editor.jsx");
 var EnabledFeatures = require("./enabled-features.jsx");
 var JsonEditor = require("./json-editor.jsx");
 var Renderer = require("./renderer.jsx");
+var Util = require("./util.js");
 
 var rendererProps = React.PropTypes.shape({
     content: React.PropTypes.string,
     widgets: React.PropTypes.object,
-    images: React.PropTypes.object,
+    images: React.PropTypes.object
```

Comment:
nit: All of the browsers we support (IE9+) support trailing commas now, so we can actually leave these in. Since this is pretty recent (we dropped support for IE8 last summer) our code is pretty inconsistent, but I've been moving towards using trailing commas everywhere. They make diffs and moving lines nicer, after one gets over the weird looks.